src/Config/Converters/EntityGraphQLOptionsConverter.cs

```diff
@@ -0,0 +1,128 @@
+                }
+
+                if (reader.TokenType == JsonTokenType.PropertyName)
+                {
+                    string? property = reader.GetString();
+                    reader.Read();
+
+                    switch (property)
+                    {
+                        case "enabled":
+                            enabled = reader.GetBoolean();
+                            break;
+                        case "type":
+                            if (reader.TokenType == JsonTokenType.String)
+                            {
+                                singular = reader.DeserializeString() ?? string.Empty;
+                            }
+                            else if (reader.TokenType == JsonTokenType.StartObject)
+                            {
+                                while (reader.Read())
+                                {
+                                    if (reader.TokenType == JsonTokenType.EndObject)
+                                    {
+                                        break;
+                                    }
+
+                                    if (reader.TokenType == JsonTokenType.PropertyName)
+                                    {
+                                        string? property2 = reader.GetString();
+                                        reader.Read();
+                                        switch (property2)
+                                        {
+                                            case "singular":
+                                                singular = reader.DeserializeString() ?? string.Empty;
+                                                break;
+                                            case "plural":
+                                                plural = reader.DeserializeString() ?? string.Empty;
+                                                break;
+                                        }
+                                    }
+                                }
+                            }
+
+                            break;
+
+                        case "operation":
+                            string? op = reader.DeserializeString();
+
+                            if (op is not null)
+                            {
+                                operation = Enum.Parse<GraphQLOperation>(op, ignoreCase: true);
+                            }
+
+                            break;
+                    }
+                }
+            }
+        }
+
+        if (reader.TokenType == JsonTokenType.True)
+        {
+            return new EntityGraphQLOptions(Singular: string.Empty, Plural: string.Empty, Enabled: true);
+        }
+
+        if (reader.TokenType == JsonTokenType.False || reader.TokenType == JsonTokenType.Null)
+        {
+            return new EntityGraphQLOptions(Singular: string.Empty, Plural: string.Empty, Enabled: false);
```

Comment:
Looks like while deserializing we dont pluralize the name if not provided but we do so while serializing? As per Cli/Utils.cs: ConstructGraphQLTypeDetails()
Why the discrepancy?

Reply:
It was mimicking the original functionality of that method, but in reality it isn't needed, as the pluralisation will happen if there's nothing set when the entity is added to the `RuntimeEntities` instance.

I've removed that logic from the Utils class and ensured the tests still pass.